src/main/java/be/minelabs/block/blocks/QuantumfieldBlock.java

```diff
@@ -42,10 +43,19 @@ public QuantumfieldBlock() {
         this.setDefaultState(getDefaultState().with(AGE, 0).with(MASTER, false));
     }
 
-    public boolean isMaster(BlockState state) {
+    public static boolean isMaster(BlockState state) {
         return state.get(MASTER);
     }
 
+    public static int getAge(BlockState state){
+        return state.get(AGE);
+    }
```

Comment:
`getAge(..)` is never used. Can we remove it then ?
`isMaster` became static, why? The method is used on instances of the block (blockstates are instances of the Block with different properties). There is no reason for it to be static.

Reply:
> getAge(..) is never used. Can we remove it then ?

There, now it gets used

> isMaster became static, why? The method is used on instances of the block (blockstates are instances of the Block with different properties). There is no reason for it to be static.

There is no reason for it not to be static :). The only difference is that if it were to be called from outside the class now we can type this: `QuantumFieldBlock.isMaster(state)` instead of this `Blocks.WEAK_BOSON_QUANTUMFIELD.isMaster(state)`.

Since the method does not depend on the instance we are able to make it static and avoid confusion that it might take specific information per quantumfield into account.  Blockstates are not instances of Block by the way.